app/assets/stylesheets/components/overview.scss

```diff
@@ -1,5 +1,5 @@
 .summary-background {
-  background: govuk-colour("light-blue");
+  background: #005ea5;
```

Comment:
I think this is `$govuk-brand-colour` (https://design-system.service.gov.uk/styles/colour/). Not exactly sure how you reference it here, but think we should.